site/static/error.html

```diff
@@ -1,187 +1,181 @@
-            aria-hidden="true"
-            viewBox="0 0 24 24"
-          >
-            <path
-              fill="#e66828"
-              d="M12 5.99L19.53 19H4.47L12 5.99M12 2L1 21h22L12 2zm1 14h-2v2h2v-2zm0-6h-2v4h2v-4z"
-            ></path>
-          </svg>
-          <h3>Warnings</h3>
-        </div>
-        <ul>
-          {{ range $i, $v := .Error.Warnings }}
-          <li>{{ $v }}</li>
-          {{ end }}
-        </ul>
-      </div>
-      {{ end }}
-      <div class="button-group">
-        {{- if and .Error.AdditionalButtonText .Error.AdditionalButtonLink }}
-        <a href="{{ .Error.AdditionalButtonLink }}"
-          >{{ .Error.AdditionalButtonText }}</a
-        >
-        {{ end }} {{- if .Error.RetryEnabled }}
-        <button onclick="window.location.reload()">Retry</button>
-        {{ end }}
-        <a href="{{ .Error.DashboardURL }}">Back to site</a>
-      </div>
-    </div>
-  </body>
+
+<head>
+	<meta charset="UTF-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>
+		{{- if not .Error.HideStatus }}{{ .Error.Status }} - {{end}}{{
+		.Error.Title }}
+	</title>
+	<style>
+		* {
+			padding: 0;
+			margin: 0;
+			box-sizing: border-box;
+		}
+
+		html,
+		body {
+			background-color: #05060b;
+			color: #f7f9fd;
+			display: flex;
+			align-items: center;
+			justify-content: center;
+			font-family: sans-serif;
+			font-size: 16px;
+			height: 100%;
+		}
+
+		.container {
+			--side-padding: 24px;
+			width: 100%;
+			max-width: calc(500px + var(--side-padding) * 2);
+			padding: 0 var(--side-padding);
+			text-align: center;
+		}
+
+		.coder-svg {
+			width: 80px;
+			margin-bottom: 24px;
+		}
+
+		h1 {
+			font-size: 24px;
+			margin-bottom: 8px;
+			font-weight: 400;
+		}
+
+		p,
+		li {
+			color: rgb(161, 161, 170);
+			line-height: 22.4px;
+		}
+
+
+		.button-group {
+			display: flex;
+			align-items: center;
+			justify-content: center;
+			gap: 12px;
+			margin-top: 24px;
+		}
+
+		.button-group a,
+		.button-group button {
+			display: inline-flex;
+			align-items: center;
+			justify-content: center;
+			padding: 6px 16px;
+			border-radius: 4px;
+			border: 1px solid #2c3854;
+			text-decoration: none;
+			background: none;
+			font-size: inherit;
+			color: inherit;
+			width: 200px;
+			height: 42px;
+			cursor: pointer;
+		}
+
+		.button-group a:hover,
+		.button-group button:hover {
+			border-color: hsl(222, 31%, 40%);
+		}
+
+		.warning {
+			margin-top: 24px;
+			border: 1px solid rgb(243, 140, 89);
+			background: rgb(13, 19, 33);
+			width: 100%;
+			padding: 24px;
+			text-align: left;
+		}
+
+		.warning-title {
+			display: inline-flex;
+			align-self: center;
+			align-items: center;
+		}
+
+		.svg-icon svg {
+			height: 1em;
+			width: 1em;
+		}
+
+		.warning-title h3 {
+			margin-left: 10px;
+			font-size: 16px;
+			font-weight: 500;
+		}
+
+		.warning li {
+			text-align: left;
```

Comment:
where is the center align coming from? manually left aligning things that should just be left aligned feels like a bit of a bandaid